tests/templates/test_subroutines/test_qsvt.py

```diff
@@ -439,30 +433,20 @@ def test_qsvt_tensorflow(self, input_matrix, angles, wires):
         assert np.allclose(qml.matrix(op), default_matrix)
         assert qml.math.get_interface(qml.matrix(op)) == "tensorflow"
 
-    @pytest.mark.parametrize(
-        ("A", "phis", "wires"),
-        [
-            (
-                [[0.1, 0.2], [0.3, 0.4]],
-                [0.1, 0.2, 0.3],
-                [0, 1],
-            )
-        ],
-    )
-    def test_qsvt_grad(self, A, phis, wires):
+    def test_qsvt_grad(self):
         """Test that qml.grad results are the same as finite difference results"""
 
         @qml.qnode(qml.device("default.qubit", wires=2))
         def circuit(A, phis):
             qml.qsvt(
                 A,
                 phis,
-                wires=wires,
+                wires=[0, 1],
             )
             return qml.expval(qml.PauliZ(wires=0))
 
-        A = np.array([[0.1j, 0.2], [0.3, 0.4]], dtype=complex, requires_grad=True)
-        phis = np.array([0.1, 0.2, 0.3], requires_grad=True)
+        A = np.array([[0.1, 0.2], [0.3, 0.4]], dtype=complex, requires_grad=True)
+        phis = np.array([0.1, 0.2, 0.3], dtype=complex, requires_grad=True)
```

Comment:
Why this change?

Reply:
Just to make sure it handles complex types :+1: 
Let me know your thoughts if it doesn't make sense.